src/Api/Movies/MoviesApi.tsx

```diff
@@ -0,0 +1,51 @@
+const serverUrl = "http://18.60.129.53/";
+
+export const getMovies = async (value: number) => {
```

Comment:
```suggestion
export const getMovies = async (noOfMovies: number) => {
```

Reply:
Renamed the parameter as suggested in the latest commit